### DISCUSSION TITLE
Using Transformer for sensitive data masking with Fake data generation option or avoiding or templating some values

### DISCUSSION THREAD

**Discussion by YuriyGavrilov** [apache/seatunnel#7746] [Ideas]
Hi All, 

So there in an idea and goal to source and sink completely full postgres to another postgres with data masking or generation fake data for sensitive attributes. Good to know that there are a lot of fakesource available with random generators but at this moment I don't know is it working in transformer or not. Also some good news that there is dynamic compilation available for some completely custom cases.

Some maybe will try to use Transformer in case of masking ad fake generation.  

The real case is to make data synchronization from prod to test environment with some predefined option by user request

What do you think?

**Comment by Hisoka-X** [comment]
+1

**Reply by Hisoka-X** [reply]
please create an issue for this feature request.

**Comment by YuriyGavrilov** [comment]
👍 @Hisoka-X https://github.com/apache/seatunnel/issues/7766
